src/index.tsx

```diff
@@ -95,6 +105,7 @@ export function useSpeechRecognition(options: SpeechRecognitionOptions = default
   }, [status, recognition, resetTranscript]);
 
   const stopListening = useCallback(() => {
+    options.onEnd && options.onEnd();
```

Comment:
Which event does this correlate to in the `SpeechRecognition` API? Can we add it directly on there instead of a manual trigger here or is it not covered by one of those events?

Reply:
Yes the SpeechRecognition API has an .onend event but it is set like this in the code: `recognition.onend = onRecognitionEnd;`, and `onRecognitionEnd` just makes some decision to know whether to stop or keep the speech recognition, but I also noticed that the speech recognition truly stops when the stopListening method is executed and that's why I placed that options.onEnd on the stopListening method